spec/granite/form/model/associations/embeds_many_spec.rb

```diff
@@ -342,37 +367,46 @@
     specify { expect(association.writer([])).to eq([]) }
 
     specify { expect(association.writer([new_project1])).to eq([new_project1]) }
+
     specify do
       expect { association.writer([new_project1]) }
-        .to change { association.reader.map(&:attributes) }.from([]).to([{'title' => 'Project 1'}])
+        .to change { association.reader.map(&:attributes) }.from([]).to([{ 'title' => 'Project 1' }])
     end
 
     specify do
       expect { existing_association.writer([new_project1, Dummy.new, new_project2]) }
         .to raise_error Granite::Form::AssociationTypeMismatch
     end
+
     specify do
-      expect { muffle(Granite::Form::AssociationTypeMismatch) { existing_association.writer([new_project1, Dummy.new, new_project2]) } }
-        .not_to change { existing_association.reader }
+      expect do
+        muffle(Granite::Form::AssociationTypeMismatch) do
+          existing_association.writer([new_project1, Dummy.new, new_project2])
+        end
+      end
+        .not_to(change { existing_association.reader })
```

Comment:
It's hard to read, could it be combined with the previous one as `.to raise_error ... .and(not_to(change ...))`?

Reply:
The previous one doesn't do `muffle`. I've removed the one line here, I don't think we can improve much more without diving deep into the spec